packages/design-system/src/components/Tooltip/Tooltip.jsx

```diff
@@ -108,30 +129,37 @@ export class Tooltip extends React.Component {
     const TriggerComponent = this.triggerComponentType();
     const triggerClasses = classNames('ds-base', 'ds-c-tooltip__trigger', triggerClassName, {
       [triggerActiveClassName]: this.state.active,
-      'ds-c-tooltip__trigger--click-only-active': this.props.dialog && this.state.active,
```

Comment:
This styling was removed, no longer necessary with some of the event handler changes